ts/data/case.ts

```diff
@@ -191,14 +191,22 @@ export const caseDefinition: TableDefinition<Case, Test> = {
     legendAction,
     {
       title: "TAT Report",
-      handler: (filters: { key: string; value: string }[]) => {
+      handler: (
+        filters: { key: string; value: string }[],
+        baseFilter: { key: string; value: string } | undefined
+      ) => {
         const currentFilters: { [key: string]: string[] } = {};
+
+        if (baseFilter !== undefined) {
+          currentFilters[baseFilter.key] = baseFilter.value.split(",");
+        }
+
         for (const filter of filters) {
           if (filter.value !== undefined && filter.value !== null) {
             if (!currentFilters[filter.key]) {
               currentFilters[filter.key] = [];
             }
-            currentFilters[filter.key].push(filter.value);
+            currentFilters[filter.key].push(...filter.value.split(","));
           }
         }
         const joinedFilters: { [key: string]: string } = {};
```

Comment:
With the change to `JsonNode` on the back-end, there's no longer a need for this manipulation. Just send the filters as `{ key: string; value: string }[]` like they already are. That makes handling them on the back-end simpler too. You will just need to create a new array that includes the base filter and all regular filters. Don't manipulate the original, as that would affect the table